frontend/src/employee-mobile-frontend/components/attendances/child-info/ArrivalAndDeparture.tsx

```diff
@@ -34,57 +34,70 @@ export default React.memo(function ArrivalAndDeparture({
     return returnToComing(unitId, child.id)
   }
 
-  const arrival = child.attendance?.arrived
-  const departure = child.attendance?.departed
+  const latestArrival =
+    child.attendances.length > 0 ? child.attendances[0].arrived : null
+  //const latestDeparture = child.attendances.length > 0 ? child.attendances[0].departed : null
 
-  if (!arrival) {
+  if (!latestArrival) {
     return null
   }
 
-  const arrivalDate = arrival.toLocalDate()
+  const arrivalDate = latestArrival.toLocalDate()
   const dateInfo = arrivalDate.isEqual(LocalDate.todayInSystemTz())
     ? ''
     : arrivalDate.isEqual(LocalDate.todayInSystemTz().subDays(1))
     ? i18n.common.yesterday
     : arrivalDate.format('d.M.')
 
   return (
-    <FixedSpaceRow justifyContent="center">
-      {arrival ? (
-        <ArrivalTimeContainer>
-          <ArrivalTime>
-            <span>{i18n.attendances.arrivalTime}</span>
-            <span>{`${dateInfo} ${arrival.toLocalTime().format()}`}</span>
-          </ArrivalTime>
-          {!departure && (
-            <InlineWideAsyncButton
-              text={i18n.common.cancel}
-              icon={faArrowRotateLeft}
-              onClick={() => returnToComingCall()}
-              onSuccess={() => {
-                reloadAttendances()
-                navigate(-1)
-              }}
-              data-qa="cancel-arrival-button"
-            />
-          )}
-        </ArrivalTimeContainer>
-      ) : null}
-      {departure ? (
-        <ArrivalTime>
-          <span>{i18n.attendances.departureTime}</span>
-          <span>{departure.toLocalTime().format()}</span>
-        </ArrivalTime>
-      ) : null}
-    </FixedSpaceRow>
+    <ArrivalTimeContainer>
+      {child.attendances.reverse().map(({ arrived, departed }) => (
```

Comment:
For backwards compatibility the expected order from the backend is latest first, so it can be used by the old code by referring the latest element by index 0. The UI renders the values in oldest first order, so unfortunaltely both orders are needed. I vote for latest first for backwards compatibility, and thus reverse reverse is needed